guides/developer/plugin-creators/deck-plugin.md

```diff
@@ -0,0 +1,428 @@
+
+	![Run Orca Configuration](/assets/images/guides/developer/plugin-creators/run-orca-config.png)
+
+1. Run `orca` using the `RunOrca` configuration
+
+   Success output is similar to:
+
+	```bash
+	INFO 18111 --- [main] com.netflix.spinnaker.orca.Main: [] Started Main in 11.123 seconds (JVM running for 11.933)
+	```
+
+	If Orca is unable to find Redis, make sure your Minnaker VM is running and that all the Spinnaker services are ready.
+
+	You can stop running Orca after you have verified that you can successfully run it.
+
+## Build the plugin
+
+Navigate to the `pf4jStagePlugin` directory and execute:
+
+```bash
+./gradlew releaseBundle
+```
+
+The build process creates files you need in later steps:
+
+* `random-wait-orca/build/Armory.RandomWaitPlugin-orca.plugin-ref`
+* `random-wait-deck/build/dist/index.js`
+
+## Configure Orca for the plugin
+
+1. Create a top-level `plugins` directory in your Orca project
+1. Copy the `Armory.RandomWaitPlugin-orca.plugin-ref` file to the `plugins` directory
+1. Create the `orca-local.yml` file in `~/.spinnaker/` with the following contents:
+
+   ```yaml
+   spinnaker:
+     extensibility:
+       plugins:
+         Armory.RandomWaitPlugin:
+          enabled: true
+          version: 1.1.14
+          extensions:
+            armory.randomWaitStage:
+              enabled: true
+              config:
+                defaultMaxWaitTime: 20
+	```
+
+   This tells Spinnaker to enable and use the plugin.
+
+## Import the pf4jStagePlugin project into IntelliJ
+
+1. In IntelliJ, link the `pf4jStagePlugin` project to your Orca project
+
+   1. Open the **Gradle** window in your Orca project if it's not already open (**View > Tool Windows > Gradle**)
+   1. In the **Gradle** window, click the **+** sign to link your `pf4jStagePlugin` Gradle project
+   1. Navigate to your `pf4jStagePlugin` directory , select the `build.gradle` file, and click **Open**
+
+1. In the **Gradle** window, right click **orca** and click **Reimport Gradle Project**
+
+You can now run or debug Orca and the plugin using IntelliJ.
+
+## Configure Deck for the plugin
+
+1. Update the `deck/plugin-manifest.json` with the plugin information
+
+   ```json
+	[
+		{
+			"id": "Armory.RandomWaitPlugin",
+			"url": "./plugins/index.js",
+			"version": "1.1.14"
+		}
+	]
+	```
+
+	For development, the values in `id` and `version` can be any value.
+
+1. Create a `deck/plugins` directory and `symlink` `random-wait-deck/build/dist/index.js` to `deck/plugins/index.js`. For example:
+
+   ```bash
+   cd <path-to-deck>
+   ln -s <path-to-pf4jStagePlugin>/random-wait-deck/build/dist/index.js plugins/index.js
+	```
+
+## Run the plugin and Orca in IntelliJ
+
+1. Create a new build configuration
+
+   1. Click **Edit Configurations...**
+   1. In the **Run/Debug Configurations** window, click the **+** icon and then select **Application**
+   1. Fill in fields 1-6 with the following:
+
+	  1. **Name:** "Build and Test Plugin"
```

Comment:
Bullets here, rather than numbers.

Reply:
The hard-coded numbers correspond to the numbers in the picture:

![image](https://user-images.githubusercontent.com/26799583/87331796-35b5cc00-c500-11ea-99e0-54c3eb9d1531.png)

Reply:
I think Markdown interprets these as an ordered list, numbering them 1, 2, 3, even if you hardcoded 6, 7, 8. In this case, it comes out the same, but would be a problem if the image were updated.